tests/test_loaders.py

```diff
@@ -41,3 +42,80 @@ def test_load_sound_speed(all_files_dict):
     assert isinstance(svdf, DataFrame)
     assert {SP_DEPTH, SP_SOUND_SPEED} == set(svdf.columns.values.tolist())
     assert svdf.dtypes[SP_DEPTH] == float64 and svdf.dtypes[SP_SOUND_SPEED] == float64
+
+
+@pytest.fixture
+def transponder_ids() -> List[str]:
+    config = load_configuration(TEST_DATA_FOLDER / "config.yaml")
+    transponders = config.solver.transponders
+    return [t.pxp_id for t in transponders]
+
+
+@pytest.mark.parametrize(
+    "is_j2k, time_scale",
+    [
+        (True, "tt"),
+        (False, "tt"),
+    ],
+)
+def test_load_travel_times(all_files_dict, transponder_ids, is_j2k, time_scale):

```

Comment:
Could you put comments on each pieces of the code sections on what it's doing? This would allow for a clear explanation of each pieces since this is a large test.

Reply:
Done!